docs/development/release-management.md

```diff
@@ -6,22 +6,52 @@ title: Release Management
 
 This is the process we follow to publish a new release of KSD.
 
+There are two parts to the process
+
+1. Cut and Release from the koor-tech/koor master when Rook does a major release,
+   eg. When Rook 1.12 is release, we mark our major release at the same time Koor 1.12
+
+2. Our release candidate for any other release, can be a minor version diverging from Rook
+   eg: Our regular monthly/weekly release can be v1.12.1
+
 ## Merge in changes
+### Release for Major Rook/Koor Version
 
-* Include updates from Rook.
-* Include new Koor improvements that are ready to publish.
+* Pull Changes latest changes of Rook upstream to koor-tech/koor by running `Backport upstream changes (Cronjob)` github action from Koor repository.
+* A PR with Rook upstream latest changes will be opened against master, please verify their are no merge conflicts, otherwise resolve them locally and repush.
+* Include new Koor improvements PRs that are ready to publish.
 * Make sure dependencies are up to date.
+* After review and merge of this PR, along with others, we follow the Rook release guide, to make a major release.
+* After this Create a new release branch from koor-tech/koor master, the branch should be named as `release-x.xx`, eg,
+
+```
+$ git checkout -b release-1.12
+```
+* Follow the steps for triggering release builds after the above steps.
+### Release for Minor Rook/Koor Version
+
+* To the existing release branch(created at time of major release), merge Rook upstream release branch commits.
+* Merge our backport commits if any to the koor/release-${version} branch
+* Follow the steps for triggering release builds after the above steps.
+### Steps for triggering release builds
+
+  git checkout <branch> # e.g. release-1.10
+  # set to the new release
+  tag_name=<release version> # e.g., v1.10.9
+  git tag -a "$tag_name" -m "$tag_name release tag"
+  git push upstream "$tag_name"
+  git push upstream <branch> #e.g. release-1.10
```

Comment:
This needs to be wrapped in triple back ticks as the vitepress build process otherwise fails with:

```
16:04:55.087 | [vite:vue] Element is missing end tag.
-- | --
16:04:55.087 | file: /opt/buildhome/repo/docs/development/release-management.md:38:17
16:04:55.087 | ✖ building client + server bundles...
16:04:55.333 | [vite:vue] Element is missing end tag.
16:04:55.333 | file: /opt/buildhome/repo/docs/development/release-management.md:38:17
16:04:55.356 | build error:
16:04:55.356 | SyntaxError: Element is missing end tag.
16:04:55.356 | at createCompilerError (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:17:17)
16:04:55.357 | at emitError (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:1446:5)
16:04:55.357 | at parseElement (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:1022:5)
16:04:55.357 | at parseChildren (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:840:18)
16:04:55.357 | at parseElement (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:999:20)
16:04:55.357 | at parseChildren (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:840:18)
16:04:55.357 | at parseElement (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:999:20)
16:04:55.357 | at parseChildren (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:840:18)
16:04:55.358 | at parseElement (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:999:20)
16:04:55.358 | at parseChildren (/opt/buildhome/repo/node_modules/@vue/compiler-core/dist/compiler-core.cjs.prod.js:840:18)
16:04:55.386 | Failed: build command exited with code: 1
```

You can run it locally using `npm run docs:build`